packages/sveltekit/src/builder.ts

```diff
@@ -160,12 +160,8 @@ export const POST = async ({request}) => {
     // Post-process the generated file to wrap with SvelteKit request converter
     let webhookRouteContent = await readFile(webhookRouteFile, 'utf-8');
 
-    // For local dev (node), need this since context isn't available to waitUntil()
-    // Add SYMBOL_FOR_REQ_CONTEXT at the top after imports
-    webhookRouteContent = webhookRouteContent.replace(
-      /(import.*?;)/,
-      `$1\n\nconst SYMBOL_FOR_REQ_CONTEXT = Symbol.for('@vercel/request-context');`
-    );
+    webhookRouteContent = `process.on('unhandledRejection', (reason) => { if (reason !== undefined) console.error('Unhandled rejection detected', reason); });
```

Comment:
let's please add a comment about this with an explanation. preferably use "HACK:" or "NOTE:" so we can address this later. Otherwise it will get lost and we'll forget context

Reply:
cc @ijjk  for vis